src/client/src/services/mappers/tradeMapper.js

```diff
@@ -10,12 +10,18 @@ export default class TradeMapper {
   }
 
   mapFromDto(dto:Object) : Array<Trade> {
-    return _.map(
+    let tradesUpdate = new TradesUpdate(dto.IsStateOfTheWorld, dto.IsStale);
```

Comment:
this should take the trades in it's ctor, typically service domain entities are 'immutable' so should not be altered after creation.